docs/installation/installation-detailed.rst

```diff
@@ -22,8 +22,7 @@ Detailed installation guide
 
 .. admonition:: CAVEAT LECTOR
 
-   Invenio v3.0 alpha is a bleeding-edge developer preview version that is
-   scheduled for public release in Q3/2016.
+   Invenio v3.0 alpha is a bleeding-edge developer preview version.
```

Comment:
OK. We do have a [roadmap](https://github.com/inveniosoftware/invenio/wiki/Invenio-3.0-Roadmap), but  since the work is taken up mostly by "volunteering", ETAs are hard indeed.